lib/Command/TransferOwnership.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace OCA\Deck\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class TransferOwnership extends Command {
+
+	protected function configure() {
+		$this
+			->setName('deck:transfer-ownership')
+			->setDescription('Change owner of deck entities')
+			->addArgument(
+				'owner',
+				InputArgument::REQUIRED,
+				'Owner uid'
+			)
+            ->addArgument(
+                'newOwner',
+                InputArgument::REQUIRED,
+                'New owner uid'
+            );
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$owner = $input->getArgument('owner');
+		$newOwner = $input->getArgument('newOwner');
+        $db = \OC::$server->getDatabaseConnection();
```

Comment:
Please use dependency injection in the constructor to get the required class instances.

Reply:
Outdated